app/src/main/res/values/strings.xml

```diff
@@ -596,23 +596,23 @@
     <string name="about_sponsor">sponsor</string>
     <string name="about_sponsor_link_text">Sponsor</string>
     <string name="about_app">app</string>
-    <string name="about_email">Email us</string>
+    <string name="about_email">E-mail us</string>
     <string name="about_settings">system</string>
     <string name="about_settings_app_info">App info</string>
-    <string name="about_settings_vpn_profile">VPN Profile</string>
-    <string name="about_settings_notification">Notification Settings</string>
+    <string name="about_settings_vpn_profile">VPN profile</string>
+    <string name="about_settings_notification">Notification settings</string>
     <string name="about_whats_new">What\'s new in %1$s</string>
     <string name="about_twitter">Follow us on Twitter</string>
```

Comment:
```suggestion
    <string name="about_twitter">Twitter</string>
```

Reply:
It is meant as a call-to-action aka `Follow us on Twitter` is just about right for that...

Reply:
"Circumvent censorship … on your Android device."
Live by the sword.